tests/integration/test_k8s.py

```diff
@@ -41,10 +40,10 @@ async def ready_nodes(k8s, expected_count):
         )
         for node in node_list["items"]
     }
-    log.info(f"Found {len(nodes)} nodes...")
+    log.info("Found %d/%d nodes...", len(nodes), expected_count)
     assert len(nodes) == expected_count, f"Expect {expected_count} nodes in the list"
     for node, ready in nodes.items():
-        log.info(f"Node {node} is not yet ready...")
+        log.info("Node %s is %sready...", node, "" if ready else "not ")
```

Comment:
How about log.info(f"Node {node} is {'ready' if ready else 'not ready'}...") ?

Reply:
sure why not... just without the `f-strings`

Reply:
```suggestion
        log.info("Node %s is %s..", node, "ready" if ready else "not ready")
```